Levels/NewStarbase/NewStarbase.prefab

```diff
@@ -30814,7 +30844,7 @@
                 {
                     "op": "replace",
                     "path": "/ContainerEntity/Components/Component_[2563122736071885678]/Transform Data/Rotate/1",
-                    "value": 1.2738307587564779e-12
+                    "value": 1.273830758756478e-12
```

Comment:
nit: Do we know why these changed?

Reply:
nope. I didn't change anything in the level aside from adding the comment components;